src/amchart/Column.js

```diff
@@ -8,68 +8,56 @@
 }(this, function(d3, CommonSerial, AmCharts, INDChart) {
     function Column() {
         CommonSerial.call(this);
-
         this._tag = "div";
         this._gType = "column";
-
         this.orientation("horizontal");
     }
     Column.prototype = Object.create(CommonSerial.prototype);
     Column.prototype.constructor = Column;
     Column.prototype._class += " amchart_Column";
     Column.prototype.implements(INDChart.prototype);
-
     Column.prototype.publish("paletteID", "default", "set", "Palette ID", Column.prototype._palette.switch(),{tags:["Basic","Shared"]});
     Column.prototype.publish("stacked", false, "boolean", "Stack Chart",null,{tags:["Basic","Shared"]});
     Column.prototype.publish("fillOpacity", 0.7, "number", "Opacity of The Fill Color", null, {min:0,max:1,step:0.001,inputType:"range",tags:["Intermediate","Shared"]});
-
     Column.prototype.publish("cylinderBars", false, "boolean", "Cylinder Bars",null,{tags:["Basic"]});
     Column.prototype.publish("circleRadius", 1, "number", "Circle Radius of Cylinder Bars",null,{tags:["Basic"]});
-
     Column.prototype.publish("columnWidth", 0.62, "number", "Column Width",null,{tags:["Basic"]});
-
     Column.prototype.publish("Depth3D", 0, "number", "3D Depth (px)",null,{tags:["Basic"]});
     Column.prototype.publish("Angle3D", 0, "number", "3D Angle (Deg)",null,{tags:["Basic"]});
-
     Column.prototype.publish("stackType", "regular", "set", "Stack Type",["none","regular","100%","3d"],{tags:["Basic"]});
     Column.prototype.publish("useOhlcLines", false, "boolean", "Use OHLC Lines",null,{tags:["Intermediate"]});
-
+    Column.prototype.publish("yAxisMinimumValue", 0, "number", "Y axis Minimum value",null,{});
+    Column.prototype.publish("yAxisMaximumValue", 100, "number", "Y axis Maximum value",null,{});
     Column.prototype.enter = function(domNode, element) {
         CommonSerial.prototype.enter.apply(this, arguments);
     };
-
     Column.prototype.updateChartOptions = function() {
         CommonSerial.prototype.updateChartOptions.apply(this, arguments);
-
         if (this._rangeType === "candle-ohlc") {
             this._gType = this.useOhlcLines() ? "ohlc" : "candlestick";
         } else {
             this._gType = "column";
         }
-
         this.buildGraphs(this._gType);
-
-        // Stacked
         if(this.stacked()){
             this._chart.valueAxes[0].stackType = this.stackType();
         } else {
             this._chart.valueAxes[0].stackType = "none";
         }
-
+        this._chart.valueAxes[0].minimum = this.yAxisMinimumValue();
+        this._chart.valueAxes[0].maximum = this.yAxisMaximumValue();
+        this._chart.valueAxes[0].baseValue = 0;
+        this._chart.valueAxes[0].strictMinMax = true;
```

Comment:
With the optional params above this would then become:
```
if (this.yAxisMinimumValue_exists() && this.yAxisMaximumValue_exists()){
  this._chart.valueAxes[0].minimum = this.yAxisMinimumValue();
  this._chart.valueAxes[0].maximum = this.yAxisMaximumValue();
  this._chart.valueAxes[0].baseValue = 0;
  this._chart.valueAxes[0].strictMinMax = true;
}
```
(You may need to remove those items if the values no longer exist.